nautobot/extras/models/change_logging.py

```diff
@@ -67,6 +67,8 @@ class ObjectChange(BaseModel):
     changed_object_type = models.ForeignKey(to=ContentType, on_delete=models.PROTECT, related_name="+")
     changed_object_id = models.UUIDField(db_index=True)
     changed_object = GenericForeignKey(ct_field="changed_object_type", fk_field="changed_object_id")
+    change_context = models.CharField(max_length=50, choices=ObjectChangeEventContextChoices, blank=True)
+    change_context_detail = models.CharField(max_length=100, blank=True)
```

Comment:
`blank=True` is just here for pre-existing records only, right? Wonder if it'd be better to disallow blank values and either populate existing records as `web` or alternatively add an `unknown` value that can be used for those?